### PR TITLE
Feature/controlleras (issue #101)

### PR DESCRIPTION
**Pull request by yanivefraim**
Added `controller as`  to controller's section (issue #101)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,84 +295,110 @@ Currently there's not a big difference, but the first way looks cleaner. Also, i
...
+      {{ main.title }}
+   </div>
+   ```
+   
+   ```JavaScript
+   app.controller('MainCtrl', MainCtrl);
+   
+   function MainCtrl () {
+      this.title = 'Some title';
+   });
```
Comment (mgechev) [inline README.md]: There's extra parentheses here.

File: README-ja-jp.md [hunk 2]
```diff
@@ -262,7 +262,8 @@ module.factory('Service', function ($rootScope, $timeout, MyCustomDependency1, M
 # コントローラ
 
 * コントローラ内でDOMを操作してはいけません。テストがしづらくなりますし、[関心の分離](https://en.wikipedia.org/wiki/Separation_of_concerns)の原則を破ることになります。代わりにティレクティブを使いましょう。
-* コントローラ名は、そのコントローラの機能を表す名前(例: shopping cart, homepage, admin panel)にし、最後に `Ctrl` を付けます。UpperCamelCase(`HomePageCtrl`, `ShoppingCartCtrl`, `AdminPanelCtrl`, etc.)を使いましょう。
+* コントローラ名は、そのコントローラの機能を表す名前(例: shopping cart, homepage, admin panel)にし、最後に `Ctrl` を付けます。
```
Comment (mgechev) [inline README-ja-jp.md]: You know Japanese and this change was intentional or by mistake?

**Comment by mgechev** [comment]
Looks great! You can fix the syntax error and I'll merge it in master.

File: README-ja-jp.md (hunk 2 above)
Comment (yanivefraim) [inline README-ja-jp.md]: I do not know Japanese and I do not remember changing this... (:

Comment (mgechev) [inline README-ja-jp.md]: Ok, lets ask @morizotter for help :-)

**Comment by yanivefraim** [comment]
Fixed (removed the extra parentheses). Thanks!

File: README-ja-jp.md (hunk 2 above)
Comment (morizotter) [inline README-ja-jp.md]: These Japanese changes are not problem. I don't know why but @mgechev 's commit is included in this pr :smiley_cat:

**Pull request merged**